XamarinCommunityToolkit/Converters/InvertedBoolConverter.shared.cs

```diff
@@ -27,6 +33,12 @@ public class InvertedBooleanConverter : IValueConverter
         /// <param name="parameter">Additional parameter for the converter to handle. Not implemented.</param>
         /// <param name="culture">The culture to use in the converter.</param>
         /// <returns>An inverted boolean from the one coming in.</returns>
-        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture) => !(bool)value;
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value is bool result)
```

Comment:
Since **Convert** and **ConvertBack** do the same, we probably could extract the logic to the separate private method to satisfy DRY principle, but uh